app/models/authentication.rb

```diff
@@ -0,0 +1,11 @@
+class Authentication
+
+	def self.visit(code)
```

Comment:
@Dpalazzari sure! if it's specific to github (which it is), then that makes sense 👍